src/main/java/study/calculator/StringDelimiter.java

```diff
@@ -0,0 +1,24 @@
+package study.calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.stream.Collectors;
+
+public class StringDelimiter {
+
+    private final Delimiter delimiter;
+
+    public StringDelimiter(Delimiter delimiter) {
+        this.delimiter = delimiter;
+    }
+
+    public List<String> filteredString(String target) {
+        Matcher matcher = Delimiter.DELLIMITER_PATTERN.matcher(target);
+        if (matcher.find()) {
+            String[] split = matcher.group(2).split(delimiter.delimiter());
```

Comment:
로컬변수도 조금 더 의미 있는 이름을 부여하면 좋을 것 같아요
```suggestion
            String[] numbers = matcher.group(2).split(delimiter.delimiter());
```